aptos-move/framework/aptos-framework/sources/chain_status.spec.move

```diff
@@ -4,8 +4,13 @@ spec aptos_framework::chain_status {
         pragma aborts_if_is_strict;
     }
 
-    spec set_genesis_end {
-        pragma verify = false;
+    spec set_genesis_end(aptos_framework: &signer) {
+        use std::signer;
+        requires is_operating();
```

Comment:
I don't think it is right because is_operating is true after calling `set_genesis_end`